src/axom/core/examples/core_containers.cpp

```diff
@@ -17,7 +17,7 @@
   */
 
 // Axom includes
```

Comment:
It seems like this example file should be updated to the new `axom::Array`, where possible.

This should likely happen in a future PR after the next round of updates to `axom::Array`.